### PR TITLE
feat(calendar-weekday): weekday format is dynamic

### PR DESCRIPTION
**Pull request by marBerWei**

### CODE REVIEW

File: src/CalendarWeekHeader.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-export default function CalendarWeekHeader({ locale }) {
+export default function CalendarWeekHeader({ locale, formatWeekday }) {
   const today = new Date()
 
   const weekDays = eachDayOfInterval({
     start: startOfWeek(today, { locale }),
     end: endOfWeek(today, { locale })
-  }).map(date => format(date, 'eee', { locale }))
+  }).map(date => format(date, formatWeekday || 'eee', { locale }))
```
Comment (hernansartorio) [inline src/CalendarWeekHeader.js]: Could you instead just pass the prop here and make `'eee'` the default value of the prop (in `CalendarWeekHeader.defaultProps`)?

Comment (marBerWei) [reply src/CalendarWeekHeader.js]: Yes absolutely! Changes coming right up. Thanks for taking a look :)

**Comment by hernansartorio** [comment]
@marBerWei perfect, thank you! One last thing (and I'm sorry to be so picky 😄), could you squash the commits into one and update the commit message to something like "Add weekday format prop" or "Allow weekday format customization"? Just for consistency with the rest of the commit history (which follows [these guidelines](https://chris.beams.io/posts/git-commit/#imperative)).

**Comment by marBerWei** [comment]
@hernansartorio please be as picky as you like! I am here to help and to learn :)

**Comment by marBerWei** [comment]
Looks like I don't have write access, so go ahead and merge when you'd like :)

**Comment by hernansartorio** [comment]
Sorry for the delay!

Oh true, merging now then :)

**Comment by hernansartorio** [comment]
And just published a new release with the change (2.0.1), thanks again!

**Pull request merged**